docs/integrator/components/nodewright.md

```
@@ -57,23 +57,37 @@ Uses a set of bash scripts to do the necessary actions to bring an ubuntu worker
-Includes the setup and optimizations for a specific service, accelerator and intent. Note that while it does have if statements around the service and intent, the inclusion of nvidia-setup (which requires service) means these are not optional today and would need to be split out to properly support that. Currently tested with:
- * eks, gb200, multiNodeTraining
- * eks, gb200, inference
- * eks, h100, multiNodeTraining
- * eks, h100, inference
+Includes the setup and optimizations for a specific service, accelerator and intent. Note that while it does have if statements around the service and intent, the inclusion of nvidia-setup (which requires service) means these are not optional today and would need to be split out to properly support that.
 
- To support non-service-specific tuning (for example, h100 + inference), the nvidia-tuned package would need to be separated out, or nvidia-setup updated to support additional services or make fewer assumptions about what it is installing — it is currently opinionated towards EKS.
+The table below is generated from the recipes by `make tuning-docs` — **do not edit it by hand**. **Setup** and **Tuning** are the pinned nodewright package versions applied for each service + accelerator. **Profile** is the tuning profile accelerator when it differs from the selected accelerator (for example, `h200` and `a100` nodes use the `h100` profile), and `–` when identical. In the **Setup** and **Tuning** columns, `–` means no such package is applied for that service + accelerator. `*` means the recipe pins no value for that dimension.
 
- See [recipes/components/nodewright-customizations/manifests/tuning.yaml](https://github.com/NVIDIA/aicr/blob/main/recipes/components/nodewright-customizations/manifests/tuning.yaml)
+{/* BEGIN AICR-TUNING */}
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)